key.core/src/main/java/de/uka/ilkd/key/macros/scripts/RuleCommand.java

```diff
@@ -61,8 +61,18 @@ public void execute(AbstractUserInterfaceControl uiControl, Parameters args, Eng
         RuleApp theApp = makeRuleApp(args, state);
         Goal g = state.getFirstOpenAutomaticGoal();
 
-        if (theApp instanceof TacletApp) {
-            RuleApp completeApp = ((TacletApp) theApp).tryToInstantiate(g.proof().getServices());
+        if (theApp instanceof TacletApp tacletApp) {
+
+            ImmutableList<TacletApp> ifSeqCandidates =
+                    tacletApp.findIfFormulaInstantiations(g.sequent(), g.proof().getServices());
```

Comment:
This call fails if the `ifFormulaInstantiations` have already been instantiated.

Reply:
should be fixed by another if.